Document/0x05h-Testing-Platform-Interaction.md

```diff
@@ -42,18 +73,24 @@ The first code block defines the new permission, which is self-explanatory. The
 </activity>
 ```
 
-Once the permission `START_MAIN_ACTIVTY` has been created, apps can request it via the `uses-permission` tag in the `AndroidManifest.xml` file. Any application granted the custom permission `START_MAIN_ACTIVITY` can then launch the `TEST_ACTIVITY`.
+Once the permission `START_MAIN_ACTIVTY` has been created, apps can request it via the `uses-permission` tag in the `AndroidManifest.xml` file. Any application granted the custom permission `START_MAIN_ACTIVITY` can then launch the `TEST_ACTIVITY`. Please note `<uses-permission android:name="myapp.permission.START_MAIN_ACTIVITY"/>` must be declared before the `<application>` or an exception will occur at runtime.
```

Comment:
Do you have a reference on this, or some context? Haven't run into this issue myself before, and I can't find any immediate documentation from the Android Developer website.

Reply:
Yes, I believe since it would be in the incorrect nesting level the permission would not be granted leading to an exception when the app tries to request permission. 

Reference is: https://stackoverflow.com/questions/25135595/permission-denied-missing-internet-permission-but-permission-is-given

Reply:
@B3nac : can you test that to make sure? StackOverflow is notorious for explanations that might need a little more testing and checking...

Reply:
Well, actually, it is a bit more simple: use these 2 references:
https://developer.android.com/guide/topics/manifest/manifest-intro#filestruct and https://developer.android.com/guide/topics/permissions/overview show this as well. Please add a clear reference to these and add it to the referneces at the end. Then we are all good

Reply:
Added references with examples.